### DISCUSSION TITLE
Record/%E5%85%B3%E4%BA%8EPycharm%E8%B0%83%E8%AF%95%E5%99%A8%E7%9A%84%E7%A6%BB%E8%B0%B1bug%E5%90%8E%E8%AE%B0/

### DISCUSSION THREAD

**Discussion by giscus** [APLaS-Plus/APLaS-Plus.github.io#5] [Announcements]
# Record/%E5%85%B3%E4%BA%8EPycharm%E8%B0%83%E8%AF%95%E5%99%A8%E7%9A%84%E7%A6%BB%E8%B0%B1bug%E5%90%8E%E8%AE%B0/

事情简述pycharm的编译器程序没完善导致的离谱报错浪费我整整一天来处理结果发现根本不影响使用😅。写出这篇是为了让更多人能提前知道这个bug并不影响调试，那就看看我的受苦过程吧😵。。。 受苦过程前两天我在试着通过调试看YOLO代码结构的时候发现的，报错如下，并且不断按步进他一直卡在一个地方： 1234567891011121314Traceback (most recent call las

https://aplas-plus.github.io/Record/%E5%85%B3%E4%BA%8EPycharm%E8%B0%83%E8%AF%95%E5%99%A8%E7%9A%84%E7%A6%BB%E8%B0%B1bug%E5%90%8E%E8%AE%B0/

**Comment by 574993075** [comment]
http://lanpangyun.com/#/register?code=rpckuvBu
